starfish/test/image/test_imagestack.py

```diff
@@ -3,11 +3,14 @@
 
 from starfish.constants import Indices
 from starfish.image import ImageStack
+from starfish.intensity_table import IntensityTable
 # don't inspect pytest fixtures in pycharm
 # noinspection PyUnresolvedReferences
 from starfish.test.dataset_fixtures import (
     synthetic_intensity_table, synthetic_spot_pass_through_stack, loaded_codebook,
-    synthetic_dataset_with_truth_values, simple_codebook_json, simple_codebook_array)
+    synthetic_dataset_with_truth_values, simple_codebook_json, simple_codebook_array,
```

Comment:
Stylistically, when we get to these long lists of imported symbols, I prefer to do one per line.